Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
@nericksen I believe that you should raise the version here to 1.0.1 with the relevant changes

Reply:
@nericksen @tomer-pan agreed. Please bump version of this pack and update the relevant release note with all the changed made.

Reply:
Hi @nericksen, the release note for version 1.0.0 wont be displayed anywhere as we don't use this file (this is the reason you cant find it in other packs).
You can either discard it or add relevant information from this file to the pack description.